ceph-rook-e2e/config/definitions/ceph-orchestrator-rook-e2e.yml

```diff
@@ -0,0 +1,96 @@
+- job:
+    name: ceph-orchestrator-rook-e2e
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && jammy && x86_64
+    display-name: 'ceph: Rook Orchestrator E2E'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+      - inject:
+          properties-content: |
+            TERM=xterm
+    parameters:
+      - string:
+          name: sha1
+          description: "commit id or a refname, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          cancel-builds-on-update: true
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          white-list-labels:
+            - orchestrator
+            - rook
+          black-list-target-branches:
```

Comment:
for which releases (or branches) are we going to backport the ceph PR: https://github.com/ceph/ceph/pull/54056

Because, as it is, the script is not existing in any other branches apart from main. We can backport it for reef as well. But if this won't get backported to other release like quincy, pacific, octopus.. then we should blacklist those branches here so that this won't run on the PRs raised against those branches

Reply:
I think we should back-port to reef at least. This way we can run e2e testing when backproting other PRs, WDYT?

Reply:
yup, sounds good to me!